interview-questions.md

```diff
@@ -7,44 +7,78 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. As a developer, I am creating a Rails application with a model called Cohort that has_many students, but OOPS! I forgot to add the foreign key. How can I fix this mistake? What is the name of the foreign key? Would the foreign key be on the Cohort model or the Student model?
 
-  Your answer:
+  Your answer: cohort model runs through students id and db migrate the new foreign key into the data
 
-  Researched answer:
+  Researched answer: a change can be made by running the rails generate migration. adding a column add column students, cohort id ,integer. then running db migrate
 
 
 
 2. Which RESTful routes must always be passed params? Why?
 
-  Your answer:
+  Your answer: urls? the params must always be passed because its telling the page what information to display
 
-  Researched answer:
+  Researched answer: RESTful routes are always passed parameters because the parameters are linked to the url and helps to get that data and diplay it or perform an action
```

Comment:
Not all RESTful routes require params. Index doesn't need a param but show does. Show needs the param of an id. Generally speaking if you need to perform an action on just one item, you will need a param.